modules/standard/Path.chpl

```diff
@@ -421,4 +421,69 @@ proc file.realPath(): string throws {
   return result;
   }
 
+  /* Collapse paths such as `foo//bar`, `foo/bar/`, `foo/./bar`, and 
+   `foo/baz/../bar` into `foo/bar`.  Warning: may alter meaning of paths
+   containing symbolic links.  Similar to :proc:`normCase`, on Windows will replace
+   forward slashes.
```

Comment:
Since this code does not replace forward slashes (and we have no way to test that it would give a valid path for native Windows, since Chapel doesn't support native Windows yet), this comment should be modified to note that support is intended, but not yet implemented.  You can base that modification off of other functions in this file.